.github/workflows/ci-test.yaml

```diff
@@ -113,3 +113,26 @@ jobs:
             /tmp/indexer_db.sql
             /tmp/*.log
         if: success() || failure()  # but not if job is manually cancelled
+
+  test-e2e-regression:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Autogenerate Go code
+        run: |
+          go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.12
+          make codegen-go
+      - name: Build Go
+        run: |
+          make oasis-indexer
+      - name: Start db
+        run: |
+          make postgres
+      - name: Run e2e regression tests
+        run : |
+          make test-e2e-regression
```

Comment:
I suggest you also create and upload artifacts for easier uploading: Lines 103-115 from the non-regression e2e test. You can add `(SOME_ABS_PATH?)/tests/e2e_regression/actual` to the paths for `actions/upload-artifact`.

Reply:
good idea, updated!